keras/backend/common.py

```diff
@@ -186,3 +189,46 @@ def image_dim_ordering():
         return 'th'
     else:
         return 'tf'
+
+
+# Placed in backends.common to avoid problems with cyclic imports
+def has_kwarg(fn, name, accept_varkw=False):
```

Comment:
It checks for any arg, not just keyword arguments, so `has_arg` would be a better name.